src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/ViewDeclarationOrder/ViewDeclarationOrderTests.cs

```diff
@@ -1,23 +1,60 @@
-﻿using Acuminator.Analyzers.StaticAnalysis;
+﻿using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+using Acuminator.Analyzers.StaticAnalysis;
 using Acuminator.Analyzers.StaticAnalysis.ViewDeclarationOrder;
 using Acuminator.Tests.Helpers;
 using Acuminator.Tests.Verification;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Xunit;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Utilities;
+
 
 namespace Acuminator.Tests.Tests.StaticAnalysis.ViewDeclarationOrder
 {
     public class ViewDeclarationOrderTests : CodeFixVerifier
     {
-	    protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new ViewDeclarationOrderAnalyzer();
+	    protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>  
+			new PXGraphAnalyzer(CodeAnalysisSettings.Default
+													.WithRecursiveAnalysisEnabled()
+													.WithIsvSpecificAnalyzersEnabled(), 
```

Comment:
Is PX1004/PX1006 analyzer affected by ISV-Only options?

Reply:
No, because it was one of the first diagnostic from the hackathon. We can add this if we want, but it should be documented

Reply:
Then it should not be here. The default behavior is fine, and the test doesn't require specifying this explicitly.

Reply:
ok